tests/test_unit/test_builtin_modules/test_cdf_auth_readwrite_all.py

```diff
@@ -1,129 +0,0 @@
-from pathlib import Path
-from typing import Any
-
-import pyperclip
```

Comment:
Lets drop pyperclip as a dependency in the pyproject.toml (ok to save for another PR) as it is not used anymore